src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Models/DependencyModel.cs

```diff
@@ -78,7 +78,7 @@ public virtual string Id
             {
                 if (_id == null)
                 {
-                    _id = $"{OriginalItemSpec}\\{Version}".TrimEnd('\\');
+                    _id = $"{OriginalItemSpec}\\{Version}".TrimEnd(CommonConstants.BackSlashDelimiter);
```

Comment:
This is another case where .net core has added an overload that avoids allocation. Maybe we should track these for if/when the APIs get ported back...